### ISSUE TITLE
Copy/Paste in email body (HTML mode) wont work with keyboard shortcuts in Firefox.

### ISSUE THREAD

**Issue by Github-Citizen** [roundcube/roundcubemail#7928]
I had this problem in RC 1.4.10 and upgraded to 1.4.11 but the issue is still there.

In Firefox v86.0 i can not use the keyboard keys (Shift+Ins or Ctrl-V) to paste text into the email body while in HTML mode. If i use plain text mode it will work. Or i can use the mouse context menu to paste text. And to clarify im not talking about a formatting issue, no text of any kind will paste using keyboard. I tested this using Edge browser and it works using the keyboard, so it is only a Firefox thing.

Also Firefox reports the following warnings in the debug console:

`MouseEvent.mozPressure is deprecated. Use PointerEvent.pressure instead.`

`This page uses the non standard property “zoom”. Consider using calc() in the relevant property values, or using “transform” along with “transform-origin: 0 0”.`

**Comment by alecpl** [comment]
Works for me with the same Firefox version. These console warnings are unrelated. Maybe some Firefox extension is interfering? Or some Roundcube plugin?

**Comment by Github-Citizen** [comment]
I only have one FF extension for proxy, i disabled it to test and it still has the issue when disabled.
![ff_ext](https://user-images.githubusercontent.com/80302627/110473171-3cefa280-80ac-11eb-8a34-0d1dc09d87fd.png)

As for RC plugins im using:
$config['plugins'] = array(
    'password',
    'identity_select',
    'zipdownload',
);

Im willing to test/trouble shoot any idea you have.

**Comment by alecpl** [comment]
Does it depend on the place the cursor is positioned? E.g. test on an empty paragraph and inside a word.

**Comment by Github-Citizen** [comment]
Does not work on empty body, does not work in between words.
Paste will work in the To or Subject input boxes.
Copy works, i can use Ctrl-C to copy text from the body and paste to another program like notepad.

**Comment by Github-Citizen** [comment]
![about](https://user-images.githubusercontent.com/80302627/110475246-a4a6ed00-80ae-11eb-92c6-2a08f8a0f1ab.png)

**Comment by Github-Citizen** [comment]
I found what is causing the issue. In FF i changed a setting in `about:config` because some websites block you from using copy/paste on web forms. I turned this setting off allowing me to be able to paste on all webpages. It appears roundcube is relying on this feature to paste. Any way to change this so roundcube can paste in HTML mode without relying on this setting being set to true? I am able to paste in plain text mode, and in the subject field with this setting false.

`dom.event.clipboardevents.enabled=false`
